### PR TITLE
RayLLM v0.3.0

### PR DESCRIPTION
**Pull request by Yard1**
Update RayLLM to v0.3.0.

**Please note that API stability is not expected until 1.0 release. This update introduces breaking changes.**

## What's changed?

* Documentation
    - Updated readme and documentation

* API & SDK
    - Updated the format of model configuration yamls.

* Backend
    - Completely replaced the text-generation-inference based backend with [vLLM](https://github.com/vllm-project/vllm) based backend. This means RayLLM now supports all models vLLM supports.
    - Improved observability and metrics.
    - Improved testing.

This update introduces breaking changes to model configuration YAMLs related to the new vLLM backend. Refer to the updated `aviary/models/README.md` file for details on the updated configuration file format.

In order to use Aviary, ensure you are using the official Docker image `anyscale/aviary:latest`.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+# Aviary - Study stochastic parrots in the wild
```
Comment (richardliaw) [inline docs/index.md:1]: probably want to remove?

File: docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Using the Aviary CLI
```
Comment (richardliaw) [inline docs/cli.md:1]: can we remove?

**Review by richardliaw — CHANGES_REQUESTED** [review]
can you revert .md changes to docs/?

**Comment by Yard1** [comment]
@richardliaw done

**Pull request merged**